radis/test/lbl/test_calc.py

```diff
@@ -785,10 +785,22 @@ def test_calc_spectrum_multiple_molecules_wstep_auto(
         databank="hitran",  # or use 'hitemp'
         verbose=verbose,
     )
+    s_just_CO = calc_spectrum(
+        wavelength_min=4165,
+        wavelength_max=5000,  # cm-1
+        isotope="1,2,3",
+        pressure=1.01325,  # bar
+        Tgas=700,  # K
+        mole_fraction={"CO": 0.1},
+        path_length=1,  # cm
+        wstep="auto",
+        databank="hitran",  # or use 'hitemp'
+        verbose=verbose,
+    )
 
     # Check calculation went fine:
     assert set(s.conditions["molecule"]) == set(["CO2", "CO"])
-    assert s.get_conditions()["wstep"] in ("N/A", 0.013)
+    assert np.isclose(s.get_conditions()["wstep"], s_just_CO.get_conditions()["wstep"])
```

Comment:
I also believe np.isclose is a better approach for assertion since values can slightly alter over time.

Reply:
I just don't get it... The test is just not failing on my machine!

Reply:
The latest commit resolved all tests except this one. there are two different values we are getting, `0.021 and 0.013`. Could it be due to the order in which the molecules are processed? 
Like mole_fraction={"CO": 0.1, "CO2": 0.1}, if CO is processed first then wstep becomes `0.013` similar to what we are getting in `s_just_CO`, and if CO2 is processed first then wstep becomes `0.021`?

Reply:
pure CO2 spectrum wstep is `0.19` so that's not it